MailCore/Cache/DraftContentManager.swift

```diff
@@ -137,13 +137,13 @@ public class DraftContentManager: ObservableObject {
         return try await loadReplyingMessage(messageReply.message, replyMode: messageReply.replyMode).body?.freezeIfNeeded()
     }
 
-    public func replaceContent(subject: String? = nil, body: String) {
+    public func replaceContent(subject: String? = nil, body: String) async {
         guard let liveDraft = try? getLiveDraft() else { return }
-        guard let parsedMessage = try? SwiftSoup.parse(liveDraft.body) else { return }
+        guard let parsedMessage = try? await SwiftSoup.parse(liveDraft.body) else { return }
```

Comment:
This one gets a bit more tricky, this func becomes async, so it runs in the collaborative task pool.
Also we deal with a `liveDraft` from realm that need to be used in the same Thread.
But at this line, we now have a potential Thread change.

I would change this method tu use a frozen (or detached) object to call `SwiftSoup.parse(liveDraft.body)`

Then later when we write the new subject in a transaction L151, that's where I would get a live object.